### PR TITLE
Correcting old (and broken) links in /aas/README.md

### PR DESCRIPTION
**Pull request by sebbader**

### CODE REVIEW

**Comment by sebbader** [comment]
@BirgitBoss, please see if this solves #13

File: README.md [hunk 1]
```diff
@@ -22,16 +22,16 @@ It is recommended to use the qualified names for the sub-namespace and the Short
 There are two possible processes to request identifiers:
 
 1. Work with the GitHub Repository
-- Fork the [GitHub Repo](https://github.com/sebbader/id/)
+- Fork the [GitHub Repo](https://github.com/admin-shell-io/id/)
 - Add a new folder with the sub-namespace as its name
-- Create a README following the template in https://github.com/sebbader/id/template/README.md
+- Create a README following the template in https://github.com/admin-shell-io/id/templates/README.md
```
Comment (BirgitBoss) [inline README.md:27]: does not work, probably branch name is missing in path?

File: README.md [hunk 2]
```diff
@@ -22,16 +22,16 @@ It is recommended to use the qualified names for the sub-namespace and the Short
...
 - Create a Pull Request from your forked repository to the original
 
 2. Contact the [Coordination Board for Submodels](mailto:coordination-board@admin-shell.io)
 - Explain the purpose of the new identifier(s)
-- Attach the filled [template](https://github.com/sebbader/id/template/README.md)
+- Attach the filled [template](https://github.com/admin-shell-io/id/templates/README.md)
```
Comment (BirgitBoss) [inline README.md:32]: see above: does not work, probably branch name is missing in path?

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
please check path to template again, probably branch name missing? Thanks

**Comment by sebbader** [comment]
Thanks for reviewing, @BirgitBoss! Now it's fixed.

**Pull request merged**